runironic.sh

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/bash
 /usr/bin/python2 /usr/bin/ironic-conductor > /var/log/ironic-conductor.out 2>&1 &
 /usr/bin/python2 /usr/bin/ironic-api > /var/log/ironic-api.out 2>&1 &
```

Comment:
We need a way to split this out. Or even better, just run it under httpd as a wsgi application (which is anyway the recommended way to run ironic: https://docs.openstack.org/ironic/latest/install/install-rdo.html#configuring-ironic-api-behind-mod-wsgi)

Reply:
Yeah I think using wsgi makes sense which is how we do it with tripleo, and splitting it out will allow one set of logs from this container.  OK to do in a follow-on patch with above changes?